workflow/tests/hpxml_translator_test.rb

```diff
@@ -207,6 +207,29 @@ def test_run_simulation_timeseries_dst_and_utc
     assert_equal(1, timeseries_rows[0].select { |r| r == 'TimeUTC' }.size)
   end
 
+  def test_run_simulation_timeseries_output_variables
+    # Check that the simulation produces timeseries with request output variables
+    rb_path = File.join(File.dirname(__FILE__), '..', 'run_simulation.rb')
+    xml = File.join(File.dirname(__FILE__), '..', 'sample_files', 'base.xml')
+    command = "#{OpenStudio.getOpenStudioCLI} #{rb_path} -x #{xml} --hourly ALL --add-timeseries-output-variable 'Zone People Occupant Count' --add-timeseries-output-variable 'Zone People Total Heating Energy'"
```

Comment:
Once we address the above issue, let's remove the `--hourly ALL` part here to verify it still works.